app-starters-common/app-starters-trigger-common/src/main/java/org/springframework/cloud/stream/app/trigger/SourcePayloadProperties.java

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Artem Bilan
  */
-@ConfigurationProperties
+@ConfigurationProperties("trigger.source")
```

Comment:
there was already another @ConfigurationProperties with the trigger prefix and I didn't want to have a conflict with that.